.github/workflows/nightly-tests.yml

```diff
@@ -137,8 +137,15 @@ jobs:
           L4="Res: ${{ steps.resiliency-test.outcome }}"
           text="Link to <https://github.com/DataBiosphere/terra-workspace-manager/actions/runs/$GITHUB_RUN_ID|test run>"
           bold="$L2 | $L3 | $L4"
+          if [ "${{ job.status }}" == "success" ]; then
+            emoji=":white_check_mark:"
+          else
+            emoji=":no_entry:"
+            text="@channel $text"
```

Comment:
idea: Consider defining an alias for our team/first responders like `@wsm-devteam` and using that instead of `@channel`.